src/encoded/tests/test_indexing.py

```diff
@@ -170,7 +176,7 @@ def test_create_mapping_on_indexing(app, testapp, registry, elasticsearch):
         assert compare_against_existing_mapping(es, namespaced_index, item_type, item_record, True)
 
 
-@pytest.mark.action_fail
+@pytest.mark.broken_remotely
```

Comment:
I'm finding this distinction between broken_remotely and locally to be more pain than useful so may back out of it in a future PR but this will do for now.